src/main/kotlin/org/rust/ide/intentions/AddImportForMatchArmPatternIntention.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.codeInsight.intention.PriorityAction
+import com.intellij.codeInsight.intention.PriorityAction.Priority
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.ide.inspections.import.AutoImportFix
+import org.rust.ide.intentions.AddImportForMatchArmPatternIntention.Context
+import org.rust.lang.core.psi.RsMatchArm
+import org.rust.lang.core.psi.RsPatBinding
+import org.rust.lang.core.psi.RsPatIdent
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+import org.rust.lang.core.psi.ext.ancestorStrict
+import org.rust.lang.core.psi.ext.isAncestorOf
+
+class AddImportForMatchArmPatternIntention : RsElementBaseIntentionAction<Context>(), PriorityAction {
+
+    override fun getText(): String = "Import"
+    override fun getFamilyName(): String = "Add import for path in match arm"
+
+    override fun getPriority(): Priority = Priority.TOP
```

Comment:
It seems it shouldn't have top priority.
It's ok for `Import` quick-fix for unresolved reference where users have to do something. But in this case, we intentionally don't show any annotation for this action so, I think, it reflects our uncertainty about what users want here